cmd/drone-server/static/index.html

```diff
@@ -1,18 +1,18 @@
 <!DOCTYPE html>
-<html ng-app="drone" lang="en">
+<html ng-app="drone" ng-root="{{.root}}" lang="en">
 <head>
 
   <base href="/"/>
-  <link rel="stylesheet" href="/static/styles/reset.css" />
-  <link rel="stylesheet" href="/static/styles/fonts.css" />
-  <link rel="stylesheet" href="/static/styles/alert.css" />
-  <link rel="stylesheet" href="/static/styles/blankslate.css" />
-  <link rel="stylesheet" href="/static/styles/list.css" />
-  <link rel="stylesheet" href="/static/styles/label.css" />
-  <link rel="stylesheet" href="/static/styles/range.css" />
-  <link rel="stylesheet" href="/static/styles/switch.css" />
-  <link rel="stylesheet" href="/static/styles/main.css" />
-  <link rel="icon" href="/static/favicon.png">
+  <link rel="stylesheet" href="{{.root}}static/styles/reset.css" />
+  <link rel="stylesheet" href="{{.root}}static/styles/fonts.css" />
+  <link rel="stylesheet" href="{{.root}}static/styles/alert.css" />
+  <link rel="stylesheet" href="{{.root}}static/styles/blankslate.css" />
+  <link rel="stylesheet" href="{{.root}}static/styles/list.css" />
+  <link rel="stylesheet" href="{{.root}}static/styles/label.css" />
+  <link rel="stylesheet" href="{{.root}}static/styles/range.css" />
+  <link rel="stylesheet" href="{{.root}}static/styles/switch.css" />
+  <link rel="stylesheet" href="{{.root}}static/styles/main.css" />
+  <link rel="icon" href="{{.root}}static/favicon.png">
```

Comment:
I am not sure about any side effects but wouldn't `<base href="{{.root}}" />` be a bit better here?

http://www.w3schools.com/tags/att_base_href.asp

Just my 2 cent. Only randomly saw this pullrequest and thought I would comment. I am not involved in the drone development. So please only listen to the ones who are :P